Brazilian Portuguese Language Support 🇧🇷 (#553)
InnerSourceCommons/innersourcecommons.org: content/pt-br/about/_index.md

---
title: "Sobre a InnerSource Commons"
subtitle: ""
description: "A InnerSource Commons é uma comunidade em crescimento de profissionais com o objetivo de criar e compartilhar conhecimento sobre InnerSource."
draft: false
---

<section class="section">
  <div class="container">
    <div class="row align-items-center">
      <div class="col-md-4 mb-4 mb-md-0">
        {{< announcements >}}
      </div>
      <div class="col-md-7">
        <p>{{< about-text >}}
        </p>
        <a href="/about/announcements" class="btn-link">Anúncios <i class="ti-arrow-right"></i></a>
      </div>
    </div>
  </div>
</section>

<section class="section bg-light">
  <div class="container">
    <div class="row text-right align-items-center">
      <div class="col-md-6">
        <p class="section-title h2">Missão da InnerSource Commons</p>

>         <p>Nossa missão é estabelecer um corpo de conhecimento e educar indivíduos, organizações comerciais e sem fins lucrativos, centros de pesquisa e outras instituições sobre a adoção bem-sucedida das melhores práticas do InnerSource.</p>

Reviewer: <p>Nossa missão é estabelecer uma base de conhecimento e educar indivíduos, organizações com e sem fins lucrativos, centros de pesquisa e outras instituições sobre a adoção bem-sucedida das melhores práticas de InnerSource.</p>

>         <p>Nossa missão é estabelecer um corpo de conhecimento e educar indivíduos, organizações comerciais e sem fins lucrativos, centros de pesquisa e outras instituições sobre a adoção bem-sucedida das melhores práticas do InnerSource.</p>

Author: ```suggestion
    <p>Nossa missão é estabelecer uma base de conhecimento e educar indivíduos, organizações com e sem fins lucrativos, centros de pesquisa e outras instituições sobre a adoção bem-sucedida das melhores práticas de InnerSource.</p>
```

>         <p>Nós fornecemos um ambiente seguro para acumular e compartilhar conhecimento e experiências, encontrar melhores práticas, estabelecer padrões e ferramentas, produzir materiais educacionais e promover pesquisas para avançar a compreensão, adoção e prática do InnerSource.

Reviewer: <p>Oferecemos um ambiente seguro para acumular e compartilhar conhecimentos e experiências, encontrar as melhores práticas, criar padrões e ferramentas, produzir materiais educacionais e promover pesquisas para avançar na compreensão, adoção e prática de InnerSource.

>         <p>Nós fornecemos um ambiente seguro para acumular e compartilhar conhecimento e experiências, encontrar melhores práticas, estabelecer padrões e ferramentas, produzir materiais educacionais e promover pesquisas para avançar a compreensão, adoção e prática do InnerSource.

Author: ```suggestion
    <p>Oferecemos um ambiente seguro para acumular e compartilhar conhecimentos e experiências, encontrar as melhores práticas, criar padrões e ferramentas, produzir materiais educacionais e promover pesquisas para avançar na compreensão, adoção e prática de InnerSource.```

        </p>
      </div>
      <div class="col-md-6 mt-4 mb-4 mb-md-0 float-right">
        <img src="/images/about/illustrations/creative.png" class="img-fluid pl-4 pr-4">
      </div>
    </div>
  </div>
</section>

<section class="section">
  <div class="container">
    <div class="row align-items-center">
      <div class="col-md-5 mb-4 mb-md-0">
       <img src="/images/community/connection.png" class="img-fluid pl-4 pr-4">
      </div>
      <div class="col-md-6">
          <p class="section-title h2">Como Trabalhamos</p>

>           <p>A InnerSource Commons utiliza métodos de código aberto para fornecer às organizações que praticam InnerSource um fórum para discutir e aprimorar a prática do InnerSource por meio do compartilhamento de experiências (sob a <a href="https://www.chathamhouse.org/about-us/chatham-house-rule">Regra de Chatham House</a>), criação e revisão de padrões do InnerSource e troca aberta de ideias.</p>

Reviewer: A InnerSource Commons utiliza métodos de Open Source para fornecer às organizações que praticam o InnerSource um fórum para discutir e melhorar a prática do InnerSource por meio do compartilhamento de experiências (sob a Regra de Chatham House), criação e revisão dos padrões de InnerSource e a troca aberta de ideias.

>           <p>A InnerSource Commons utiliza métodos de código aberto para fornecer às organizações que praticam InnerSource um fórum para discutir e aprimorar a prática do InnerSource por meio do compartilhamento de experiências (sob a <a href="https://www.chathamhouse.org/about-us/chatham-house-rule">Regra de Chatham House</a>), criação e revisão de padrões do InnerSource e troca aberta de ideias.</p>

Author: ```suggestion
          <p>A InnerSource Commons utiliza métodos de Open Source para fornecer às organizações que praticam o InnerSource um fórum para discutir e melhorar a prática do InnerSource por meio do compartilhamento de experiências (sob a Regra de Chatham House), criação e revisão dos padrões de InnerSource e a troca aberta de ideias.</p>
```

>         <p>Indivíduos e empresas interessados são bem-vindos a participar e contribuir com conteúdo para o ISC. Em geral, o conteúdo contribuído será disponibilizado sob a <a href="https://creativecommons.org/licenses/by-sa/4.0/">Licença Internacional Creative Commons Attribution-ShareAlike 4.0</a> atribuída a InnerSourceCommons.org. Não são necessárias taxas para participar.</p>        

Reviewer: <p>Indivíduos e empresas interessados ​​são bem-vindos a participar e contribuir com conteúdo para o ISC. Em geral, o conteúdo da contribuição será disponibilizado sob a licença <a href="https://creativecommons.org/licenses/by-sa/4.0/">Creative Commons Attribution-ShareAlike 4.0 International</a> atribuída a InnerSourceCommons.org. Nenhuma taxa é necessária para participar.</p>

>         <p>Indivíduos e empresas interessados são bem-vindos a participar e contribuir com conteúdo para o ISC. Em geral, o conteúdo contribuído será disponibilizado sob a <a href="https://creativecommons.org/licenses/by-sa/4.0/">Licença Internacional Creative Commons Attribution-ShareAlike 4.0</a> atribuída a InnerSourceCommons.org. Não são necessárias taxas para participar.</p>        

Author: ```suggestion
    <p>Indivíduos e empresas interessados ​​são bem-vindos a participar e contribuir com conteúdo para o ISC. Em geral, o conteúdo da contribuição será disponibilizado sob a licença <a href="https://creativecommons.org/licenses/by-sa/4.0/">Creative Commons Attribution-ShareAlike 4.0 International</a> atribuída a InnerSourceCommons.org. Nenhuma taxa é necessária para participar.</p> ```

        <a href="/community/" class="btn-link">Confira como trabalhamos

 <i class="ti-arrow-right"></i></a>
        </div>
    </div>
  </div>
</section>

<section class="section bg-light">
  <div class="container">
    <div class="row text-right align-items-center">
      <div class="col-md-6">
        <p class="section-title h2">Membros da InnerSource Commons</p>

>         <p>A comunidade InnerSource Commons é aberta a todos. Alguns participantes da comunidade vão além e podem ser convidados a se tornar membros do ISC. A InnerSource Commons é uma corporação de associação, então os membros do ISC desempenham um papel semelhante aos acionistas de empresas de capital aberto.</p>

Reviewer: <p>A comunidade InnerSource Commons é aberta a todas as pessoas. Alguns participantes da comunidade vão além, superando as expectativas, e podem ser convidados a se tornar membros do ISC. A InnerSource Commons é uma associação de membros, então os membros da ISC desempenham um papel semelhante aos acionistas de empresas de capital aberto.</p>

>         <p>A comunidade InnerSource Commons é aberta a todos. Alguns participantes da comunidade vão além e podem ser convidados a se tornar membros do ISC. A InnerSource Commons é uma corporação de associação, então os membros do ISC desempenham um papel semelhante aos acionistas de empresas de capital aberto.</p>

Author: ```suggestion
    <p>A comunidade InnerSource Commons é aberta a todas as pessoas. Alguns participantes da comunidade vão além, superando as expectativas, e podem ser convidados a se tornar membros do ISC. A InnerSource Commons é uma associação de membros, então os membros da ISC desempenham um papel semelhante aos acionistas de empresas de capital aberto.</p>```

>         <p>Os membros têm direitos específicos no ISC; por exemplo, eles podem votar em eleições do conselho. Para ser elegível para a associação, a pessoa deve ser indicada por um membro atual. A associação é baseada exclusivamente no mérito, sem custo algum e restrita a indivíduos.

Reviewer: <p>Os membros têm direitos específicos na ISC; por exemplo, eles podem votar em eleições do conselho. Para ser elegível como um membro, a pessoa deve ser indicada por um membro atual. A associação é baseada exclusivamente no mérito, sem custo algum e restrita a indivíduos.</p>

>         <p>Os membros têm direitos específicos no ISC; por exemplo, eles podem votar em eleições do conselho. Para ser elegível para a associação, a pessoa deve ser indicada por um membro atual. A associação é baseada exclusivamente no mérito, sem custo algum e restrita a indivíduos.

Author: ```suggestion
        <p>Os membros têm direitos específicos na ISC; por exemplo, eles podem votar em eleições do conselho. Para ser elegível como um membro, a pessoa deve ser indicada por um membro atual. A associação é baseada exclusivamente no mérito, sem custo algum e restrita a indivíduos.
```

        </p>
        <a href="/about/members" class="btn-link">Todos os membros da InnerSource Commons <i class="ti-arrow-right"></i></a>
      </div>
      <div class="col-md-6 mt-4 mb-4 mb-md-0 float-right">
        <img src="/images/about/illustrations/notebook.png" class="img-fluid pl-4 pr-4">
      </div>
    </div>
  </div>
</section>

<section class="section">
  <div class="container">
    <div class="row align-items-center">
      <div class="col-md-5 mb-4 mb-md-0">
         <img src="/images/about/illustrations/team.png" class="img-fluid pl-4 pr-4">
      </div>
      <div class="col-md-6">
          <p class="section-title h2">Conselho e Governança</p>

>           <p>A InnerSource Commons é uma organização sem fins lucrativos 501(c)(3) governada por um conjunto de estatutos corporativos. O Conselho de Diretores estabelece as políticas e nomeia os oficiais que estabelecem e executam as políticas. O Conselho é eleito pelos membros anualmente.</p>

Reviewer: <p>A InnerSource Commons é uma organização sem fins lucrativos 501(c)(3) governada por um conjunto de estatutos corporativos. O Conselho de Administração define as políticas e nomeia os diretores responsáveis por estabelecer e executar essas políticas. O Conselho é eleito pelos membros da associação anualmente.</p>

>           <p>A InnerSource Commons é uma organização sem fins lucrativos 501(c)(3) governada por um conjunto de estatutos corporativos. O Conselho de Diretores estabelece as políticas e nomeia os oficiais que estabelecem e executam as políticas. O Conselho é eleito pelos membros anualmente.</p>

Author: ```suggestion
      <p>A InnerSource Commons é uma organização sem fins lucrativos 501(c)(3) governada por um conjunto de estatutos corporativos. O Conselho de Administração define as políticas e nomeia os diretores responsáveis por estabelecer e executar essas políticas. O Conselho é eleito pelos membros da associação anualmente.</p>```
